subiquity/ui/views/ubuntu_pro.py

```diff
@@ -598,16 +759,17 @@ def __init__(self, parent: UbuntuProView) -> None:
         ok = ok_btn(label=_("Continue"), on_press=lambda unused: self.close())
 
         title = _("About Ubuntu Pro")
-        header = _("Ubuntu Pro subscription gives you access to multiple"
-                   " security & compliance services, including:")
+        header = _("Ubuntu Pro is the same base Ubuntu, with an additional"
+                   " layer of security and compliance services and security"
+                   " patches covering a wider range of packages.")
 
         services = [
-            _("Security patching for over 30.000 packages, with a focus on"
-              " High and Critical CVEs (extended from 2.500)"),
-            _("10 years of security Maintenance (extended from 5 years)"),
-            _("Kernel Livepatch service for increased uptime and security"),
-            _("Ubuntu Security Guide for hardening profiles, including CIS"
-              " and DISA-STIG"),
+            _("Security patch coverage for CVSS critical, high and selected"
+              ' medium vulnerabilities in all 23,000 packages in "universe"'
```

Comment:
Would be nice to pull these numbers out of the translated string, they are likely to change.

Reply:
Yep, makes sense. Ideally, those numbers should be dependent on the version of Ubuntu being installed but I don't see a simple way to do that.